app/src/main/java/swati4star/createpdf/adapter/WhatsNewAdapter.java

```diff
@@ -0,0 +1,55 @@
+package swati4star.createpdf.adapter;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import swati4star.createpdf.R;
+
+public class WhatsNewAdapter extends BaseAdapter {
+
+    private final Context mContext;
+    private final List<String> mTitleList;
+    private final List<String> mContentList;
+    private final LayoutInflater mInflater;
+
+    public WhatsNewAdapter(Context context, ArrayList<String> titleList, ArrayList<String> contentList) {
```

Comment:
instead of different arraylist for each paramter.

create a whatsnew object & send it's arraylist as parameter